docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+# Using the Network Shim
+
+## About
+
+The Network Shim is a tool that provides a cypress plugin as well as a setup utilility for cypress' `setupFile`, which have to be used in conjunction. It uses the cypress `cy.intercept()` API to intercept network traffic during a test run. It has three different _modes_:
+
+-   `capture`: when operating in this mode, the Network Shim will intercept all outgoing requests to the specified hosts and allow each request to reach the destination server. Each response is stored as a fixture (JSON file).
+-   `stub`: in this mode the Network Shim also intercepts all outgoing requests to the specified hosts, but it will prevent the request from reaching the destination server and instead it will produce a response from a previously recorded fixture.
```

Comment:
Instead of produce:

> it will produce a response from a previously recorded fixture.

This seems easier to understand to me:

> it will reply with a response from a previously recorded fixture.

Also the sentence for this bullet point is a bit long. I'd split it up into multiple for readability.